test/uk/gov/hmrc/gform/controllers/FormTemplatesSpec.scala

```diff
@@ -44,7 +44,7 @@ class FormTemplatesSpec extends FlatSpec with Matchers with TypeclassFixtures wi
       .response(Option.empty[Schema]) // No schema will be found
       .callCheck(findOneCheck)
       .withChecks { req: JsObject =>
-        req should be(Json.obj("id" -> "http://hmrc.gov.uk/jsonschema/bf-formtemplate#"))
+        req should be(Json.obj("id" -> "http://schemaId"))
```

Comment:
There is a nicer cats syntax available after importing `import cats.syntax.all._`. Consider:
`"http:\\schemaId".some` vs `Some("http:\\schemaId")`